tasks/configs.yml

```diff
@@ -33,3 +45,15 @@
   when: "csf_allow_ip is defined"
   notify:
     - restart csf
+    
+# add ignored ip to csf.ignore
+- name: csf - adding allowed ip
+  lineinfile: >
+    dest="/etc/csf/csf.ignore"
+    regexp="^{{ item }}"
+    line="{{ item }}"
+  with_items: "{{ csf_ignore_ip|default([]) }}"
+  when: "csf_ignore_ip is defined"
```

Comment:
Aa, look in output of CI: https://travis-ci.org/jloh/csf-ansible-role/jobs/185723361
fatal: [localhost]: FAILED! => {"failed": true, "msg": "'csf_blocklists_enable' is undefined"}
I added 2 new variables: csf_blocklists_enable and csf_ignore_ip but maybe this undefined during tests?